ver/us/splat.yaml

```diff
@@ -2209,11 +2209,11 @@ segments:
     - [0x325160]
   - type: code
     start: 0x3251D0
-    vram: 0x802BD100
+    vram: 0x802C0000
     overlay: True
     subsegments:
     - [0x3251D0, c]
```

Comment:
If you add `, 3251D0` after the `c` here, you can make the next line just `- [0x3255E0]`